e2e-tests/cypress/tests/integration/channels/messaging/message_reply_bot_post_spec.js

```diff
@@ -88,7 +88,7 @@ describe('Messaging', () => {
         });
     });
 
-    it.skip('MM-T91 Replying to an older post by a user that has no content (only file attachments)', () => {
+    it('MM-T91 Replying to an older post by a user that has no content (only file attachments)', () => {
```

Comment:
Does your changes fix this one?

Reply:
yes, the problem was with the requestAnimationFrame. Cypress worked as long as you were watching it, we had kind of a quantum issue with that test :D , only worked while being observed.